presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -156,14 +161,14 @@ public IcebergSessionProperties(
                         false),
                 new PropertyMetadata<>(
                         HIVE_METASTORE_STATISTICS_MERGE_STRATEGY,
-                        "choose how to include statistics from the Hive Metastore when calculating table stats. Valid values are: "
-                                + Joiner.on(", ").join(HiveStatisticsMergeStrategy.values()),
+                        "Flags to choose which statistics from the Hive Metastore are used when calculating table stats. Valid values are: "
+                                + Joiner.on(", ").join(NUMBER_OF_DISTINCT_VALUES, TOTAL_SIZE_IN_BYTES),
```

Comment:
nitpick : Can we reference the `supportedMergeFlags` from StatisticsUtil  to build this csv list ?

Reply:
Fixed